packages/assets-controllers/src/NftController.ts

```diff
@@ -231,22 +306,30 @@ export class NftController extends BaseControllerV1<NftConfig, NftState> {
 
   private readonly messagingSystem: NftControllerMessenger;
 
-  private getNftApi({
+  getNftApi({
```

Comment:
Hmm, exposing this for another controller to use is typically something we'd avoid. Lately we've been trying to extract network calls from controllers altogether, moving them to separate "API service" classes.

But this seems OK for now, we can refactor this later.